README.md

```diff
@@ -5,11 +5,12 @@ Respond to short response questions in clear, concise sentences directly within
 
 ### Short Response Questions
 **1. What is recursion?**
-
+Recrsion is an algorithm in which it calls it self in order to break down into a smaller poblem for it to be solved in an elegant and readable way
```

Comment:
:+1: Smaller problem until what is true? 
> Recursion* is an algorithm

Algorithms can use recursion, or be recursive, maybe refer to it more as more like an approach or strategy. Our algorithms are all the steps to get there.